### ISSUE TITLE
Gem gurdiam farm was showing in the wrong place in the map

### ISSUE THREAD

**Issue by StravDrav** [Dinip/hermitcraft#134]
It should be over here
741, 1418, 64
![image](https://github.com/Dinip/hermitcraft/assets/154757846/a3b97ac4-7ed4-4db9-9374-0751f888c93b)

**Comment by Dinip** [comment]
just checked, that marker was added 20 months ago... so yeah, there are some markers that may be wrong coords
I'll fix it

**Comment by StravDrav** [comment]
Also there are some other ones that got dislocated I think
![image](https://github.com/Dinip/hermitcraft/assets/154757846/474fafc3-99ed-4f05-b60f-62ee7a591e77)

**Comment by Dinip** [comment]
Yes, the icons are not right placed, compared to the bases (only in isometric/3d view). 
Not sure what the issue is... I'll probably need to dig a bit in the code and try to understand what is causing that (the map software is not written by me, but it is publicly accessible).

I'll leave an example below, this is after I fixed the coords yesterday

![chrome_zKsv2Esz5M](https://github.com/Dinip/hermitcraft/assets/8941012/76657362-891e-43f9-ae14-47fac8fb0e82)
![chrome_so3zLPJIgB](https://github.com/Dinip/hermitcraft/assets/8941012/33c4d7d8-f942-4ab3-8443-f648be0ef927)

**Comment by StravDrav** [comment]
Got it. Anyway, you are doing a great job handling all these data so far. So good luck. 😊

**Comment by StravDrav** [comment]
dinip, do you think that this might be showing wrong because all the y positions is being a lot highest than the actual surface level? just a thought.

**Comment by Dinip** [comment]
Just to reference miclav/mapcrafter#24
